sql_generators/active_users_aggregates_v3/templates/mobile_query.sql

```diff
@@ -7,7 +7,7 @@ WITH
       adjust_network,
       install_source
     FROM
-      fenix.firefox_android_clients
```

Comment:
Should the same changes be applied across v4? For example here: https://github.com/mozilla/bigquery-etl/blob/main/sql_generators/active_users_aggregates_v4/templates/mobile_query.sql#L10